app/views/pages/home.html.erb

```diff
@@ -8,74 +8,45 @@
   </div>
 </div>
 
-<div class="card column is-three-quarters-tablet mx-auto my-5">
-  <div class="columns">
-    <div class="column is-4">
-      <div class="card-image">
-        <figure class="image is-4by3">
-          <img
-            src="https://bulma.io/images/placeholders/1280x960.png"
-            alt="Placeholder image"
-          />
-        </figure>
-      </div>
-    </div>
-    <div class="column is-8">
-      <div class="card-content p-0">
-        <div class="media">
-          <div class="media-content">
-            <p class="title is-4 m-0">Yard Sale!</p>
-            <time class="p-0" datetime="2016-1-1"> 1 Jan 2016, 07:00am</time>
-          </div>
-        </div>
-        <div class="content">
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus nec
-          iaculis mauris.
-          <br />
-          <br />
-          <div>
-            <span class="tag is-primary">Collectibles</span>
-            <span class="tag is-primary">Clothing</span>
-            <span class="tag is-primary">Electronics</span>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
+<% @yardsales.each do |yardsale| %>
```

Comment:
It may be good to have a condition to show "No yardsales available" or something like that if `@yardsales.length` == 0

Reply:
used @yardsales.any? its sooo cool and practical lol